src/components/TokenOptionItem.tsx

```diff
@@ -76,19 +76,20 @@ interface OptionItemProps {
   symbol?: string
   faded?: boolean
   warning?: string
+  url?: string
```

Comment:
`warningUrl`?, given that it's just shown if you provide the `warning` message, and it goes on the warn image